### PR TITLE
ft: S3C-1561 - add, update and delete account quotas

### PR DESCRIPTION
**Pull request by anurag4DSB**
**What does this PR do, and why do we need it?**
Add options in vaultclient to add, delete, update quotas on accounts.
- quota option when creating a new account
- update-account-quota: Update quotas on existing accounts
- delete-account-quota: Delete quotas from existing accounts

**Which issue does this PR fix?**
[S3C-1561](https://scality.atlassian.net/browse/S3C-1561)

**Special notes for your reviewers:**
ToDo after the feature is fully implemented in Vault:
- Add tests for the 3 new options. The same tests will be added in Vault, but need the options in vault client first.
- Update README for the new options

### CODE REVIEW

**Comment by anurag4DSB** [comment]
@bert-e help

**Comment by anurag4DSB** [comment]
@bert-e retry

File: lib/IAMClient.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ class VaultClient {
      * @param {string} accountName - account name
      * @param {object} options - additional creation params
      * @param {string} options.email - account email
-     * @param {string} options.password - account password
```
Comment (vrancurel) [inline lib/IAMClient.js:99]: why do you remove options.password from the doxygen ?

**Review by vrancurel — APPROVED** [review]

File: lib/IAMClient.js (hunk 1 above)
Comment (anurag4DSB) [reply lib/IAMClient.js:99]: Because we do not use it anywhere as passwords are encrypted now.

**Comment by anurag4DSB** [comment]
@bert-e approve

**Comment by anurag4DSB** [comment]
@bert-e retry

**Comment by anurag4DSB** [comment]
@bert-e status

**Pull request merged**